packages/app/src/entities/position/context/provider.tsx

```diff
@@ -65,10 +65,12 @@ export function PositionProvider({ children }: { children: JSX.Element }) {
       [],
     )
 
-    setPosition({
-      deposits: [...lendPositions] as [PositionCell, ...PositionCell[]],
-      debts: debtPositions || [],
-    })
+    if (lendPositions.length || debtPositions.length) {
```

Comment:
When we switch to another account with no positions, shouldn't we update positions to empty arrays?